module/x/gravity/keeper/attestation.go

```
@@ -128,7 +128,9 @@ func (k Keeper) TryAttestation(ctx sdk.Context, att *types.Attestation) {
 				k.emitObservedEvent(ctx, att, claim)
 
 				// Add a new bridge balance to the store, check the supply of all monitored erc20 tokens too
-				k.updateBridgeBalanceSnapshots(ctx, claim, *expectedSupplyChange)
```

Here we no longer ignore the error we spent so much time generating